mesh/nurbs.cpp

```diff
@@ -2664,7 +2913,8 @@ const
          {
             for (int i = 0; i < nx; i++)
             {
-               const int l = (kv.Size() == 2) ? p2g(i,j) : p2g(i,j,k);
+               const int ll = (kv.Size() == 2) ? p2g(i,j) : p2g(i,j,k);
+               const int l  = DofMap(l);
```

Comment:
Similar to above: these two line generate warnings: `ll` is not used and `l` is used in its own initialization.